packages/web/src/components/organisms/TransactionForm/index.tsx

```diff
@@ -85,13 +61,14 @@ export const TransactionForm = ({ propertyAddress }: Props) => {
   const handleWithdrawHolder = useCallback(() => withdrawHolder(propertyAddress), [propertyAddress, withdrawHolder])
   const handleCancelStaking = useCallback(() => {
     setIsCountingBlocks(true)
-    cancel(propertyAddress)
-      .then(checkWithdrawable)
-      .catch(() => {
-        setIsCountingBlocks(false)
-        setWithdrawable(false)
-      })
-  }, [propertyAddress, cancel, checkWithdrawable])
+    cancel(propertyAddress).catch(() => {
+      setIsCountingBlocks(false)
+      setWithdrawable(false)
+    })
+  }, [propertyAddress, cancel])
+  useEffect(() => {
```

Comment:
We don't have to await result of function in this scope, so I just used useEffect not useEffectAsync.